azure-pipelines-release.yml

```diff
@@ -6,6 +6,17 @@ pool:
   demands:
     - ImageOverride -equals MMS2022TLS
 
+variables:
+
+  - name: PackageSuffix
+    # if source branch is not `dev` then we're generating a release based on a feature branch
+    # In that case, we populate the environment variable "PackageSuffix" accordingly
+    ${{ if ne(variables['Build.SourceBranchName'], 'dev') }}:
```

Comment:
Today, this pipeline only runs when manually triggered, so it does not run automatically post-merge. The nightly pipeline does run post-merge, but that's a different `.yml`.

In this case, since this pipeline is always manual - do you think still think it makes sense to check `Build.Reason`? In my mind, the `Build.Reason` for this pipeline is always: "Manual".

Reply:
Can we get rid of the separate nightly pipeline and use this single yml and CI definition?